Cargo.toml

```diff
@@ -8,63 +8,62 @@ members = [
 default-members = ["native/acter"]
 
 [workspace.dependencies.matrix-sdk]
-git = "https://github.com/matrix-org/matrix-rust-sdk"
-rev = "1380af4f005ee5f34607db5cbb6705f93048b85a"
+git = "https://github.com/acterglobal/matrix-rust-sdk"
+rev = "5e7a05c26f7b7029a3dc4176cf758c3cc459b04f"
```

Comment:
could you elaborate what changes had to be done to the matrix sdk to switch to a local fork? And what PR is pending to get those into upstream?

Reply:
In fact, I needed `EmailIdentity::new()` and implemented it [here](https://github.com/matrix-org/matrix-rust-sdk/commit/7a2d5c5e5c88d8713e0b1dd5447f828b2efdcdfc).
But it is not useless now.
Will restore something like main branch.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1783/commits/3935bc962c9995df31b4549587d2d8d0b0fd6a1d